pkg/apis/alibabacloudprovider/v1beta1/alibabacloudmachineproviderconfig_types.go

```diff
@@ -58,186 +62,114 @@ type AlibabaCloudMachineProviderConfig struct {
-	//Passwords cannot be set for instances that run some types of operating systems such as Others Linux and Fedora CoreOS. For these instances, only key pairs can be set.
-	Password string `json:"password,omitempty"`
-
-	//The category of the system disk. Valid values:
-	//
-	//cloud_essd: ESSD. When the parameter is set to this value, you can use the SystemDisk.PerformanceLevel parameter to specify the performance level of the disk.
-	//cloud_efficiency: ultra disk.
-	//cloud_ssd: standard SSD.
-	//cloud: basic disk.
-	//For non-I/O optimized instances of retired instance types, the default value is cloud. For other instances, the default value is cloud_efficiency.
-	SystemDiskCategory string `json:"systemDiskCategory,omitempty"`
-
-	//The performance level of the ESSD used as the system disk. Default value: PL1. Valid values:
-	//
-	//PL0: A single ESSD can deliver up to 10,000 random read/write IOPS.
-	//PL1: A single ESSD can deliver up to 50,000 random read/write IOPS.
-	//PL2: A single ESSD can deliver up to 100,000 random read/write IOPS.
-	//PL3: A single ESSD can deliver up to 1,000,000 random read/write IOPS.
-	//For more information about ESSD performance levels, see ESSDs.
-	SystemDiskPerformanceLevel string `json:"systemDiskPerformanceLevel,omitempty"`
-
-	//The name of the system disk. The name must be 2 to 128 characters in length. It must start with a letter and cannot start with http:// or https://. It can contain letters, digits, colons (:), underscores (_), and hyphens (-).
-	//
-	//This parameter is empty by default.
-	SystemDiskDiskName string `json:"systemDiskDiskName,omitempty"`
-
-	//The size of the system disk. Unit: GiB. Valid values: 20 to 500.
-	//
-	//The value must be at least 20 and greater than or equal to the size of the image.
-	//
-	//The default value is 40 or the size of the image, depending on whichever is greater.
-	SystemDiskSize int `json:"systemDiskSize,omitempty"`
+	// SecurityGroups is an array of references to security groups which to assign the instance. The valid values of N vary based on the
+	// maximum number of security groups to which an instance can belong. For more information, see the "Security group limits" section in Limits.
+	// https://www.alibabacloud.com/help/doc-detail/101348.htm?spm=a2c63.p38356.879954.48.78f0199aX3dfIE
+	// +optional
+	SecurityGroups []ResourceTagReference `json:"securityGroups,omitempty"`
 
-	//The description of the system disk. The description must be 2 to 256 characters in length and cannot start with http:// or https://.
-	//
-	//This parameter is empty by default.
-	SystemDiskDescription string `json:"systemDiskDescription,omitempty"`
+	//Bandwidth describes the internet bandwidth strategy for the instance
+	// +optional
+	Bandwidth BandwidthProperties `json:"bandwidth,omitempty"`
 
-	DataDisks []DataDisk `json:"dataDisk,omitempty"`
+	//Information regarding the the system disk for the instance
```

Comment:
Nit missing space, should follow golang convention of starting with the name
```suggestion
	// SystemDisk is the configuration for the instance system disk
```

Reply:
Fixed